tests/cases/standalone/common/alter/add_col.result

```diff
@@ -59,7 +59,7 @@ DESC TABLE test;
 | i      | Int32                |     | YES  |         | FIELD         |
 | j      | TimestampMillisecond | PRI | NO   |         | TIMESTAMP     |
 | k      | Int32                |     | YES  |         | FIELD         |
-| host   | String               |     | YES  |         | FIELD         |
+| host   | String               | PRI | YES  |         | TAG           |
 +--------+----------------------+-----+------+---------+---------------+
 
 ALTER TABLE test ADD COLUMN idc STRING default 'idc' PRIMARY KEY;
```

Comment:
It's uncommon but I wanted to see what would happen if specified "not null + no default" and "not null + default"

Reply:
It's all about validations. I don't want to check them in this PR, I will create another PR for those cases.